packages/combobox/src/Combobox/Combobox.tsx

```diff
@@ -301,9 +288,12 @@ type TriggerProps = React.ComponentProps<typeof TelegraphMenu.Trigger> & {
   size?: TgphComponentProps<typeof TelegraphButton.Root>["size"];
 };
 
-const Trigger = ({ size = "1", ...props }: TriggerProps) => {
+const Trigger = ({ size = "2", ...props }: TriggerProps) => {
```

Comment:
This default was incorrectly set before when height was being improperly managed.